.github/scripts/benchmark/format-summary.py

```
@@ -0,0 +1,158 @@
+
+
+def _fmt(x):
+    if x is None:
+        return "n/a"
+    return f"{x:,.0f}" if x >= 100 else f"{x:,.1f}" if x >= 10 else f"{x:.2f}"
+
+
+def throughput_table(sizes, name_a, name_b, name_c, has_c):
+    head = ["dataset", f"A · {name_a}", f"B · {name_b}"]
+    if has_c:
+        head += [f"C · this PR", "B→C", "vs C-engine"]
```

Ruff (0.15.20)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)